dvm.sh

```diff
@@ -60,6 +72,8 @@ if [ -z "$DVM_DIR" ]; then
 fi
 unset DVM_SCRIPT_SOURCE 2> /dev/null
 
+DVM_VERSION_DIR="${DVM_DIR}/bin/docker/"
```

Comment:
Do we even have to set this here? It seems like we only need `dvm_version_path`.